contracts/zero-ex/foundry.toml

```
@@ -1,8 +1,34 @@
 [profile.default]
 src = 'contracts/src'
 out = 'foundry-artifacts'
-test = 'contracts/test/foundry'
+test = 'contracts/test/foundry/localTests'
 libs = ["contracts/deps/", "../utils/contracts/src/"]
-remappings = ['@0x/contracts-utils/=../utils/', '@0x/contracts-erc20/=../erc20/', 'src/=./contracts/src']
+remappings = [
+    'forge-std/=contracts/deps/forge-std/src/',
+    'ds-test/=contracts/deps/forge-std/lib/ds-test/src/',
+    '@0x/contracts-utils/=../utils/',
+    '@0x/contracts-erc20/=../erc20/',
+    'src/=./contracts/src',
+    '@0x/contract-addresses/=../../packages/contract-addresses',
+    'samplers/=contracts/test/samplers',
+]
 cache_path = 'foundry-cache'
 optimizer_runs = 1000000
+fs_permissions = [
+    { access = "read-write", path = "./contracts/test/foundry/addresses" },
+]
+verbosity = 3
+
+[profile.local]
+test = 'contracts/test/foundry/localTests'
```

```suggestion
test = 'contracts/test/foundry/local'
```
Naming convention got a bid inconsistent
`contracts/zero-ex/contracts/test/foundry/localTests/WrapEth.sol` is PascalCase (WrapEth) + camcelCase (localTests) and kebab-case (zero-ex).